docs/build.md

```diff
@@ -3,48 +3,58 @@
 Install packages:
 
  * build-essential
+ * debhelper
+ * debootstrap
  * sbuild
- * cpio
- * genisoimage
+ * sudo
+ * systemd-container
```

Comment:
cpio and genisoimage are required if someone wants to directly build spire, which isn't uncommon.

Reply:
Building spire requires `zip`, `python3-yaml`, and `libarchive-tools`, but `cpio` and `genisoimage` are only needed at runtime. How do you want this to be handled? Are all of spire’s build-time and runtime dependencies to be considered build prerequisites, or only its build-time dependencies, or only what’s needed to make the standard sbuild process work?

Reply:
Hmm, we probably just care about {build time dependencies} plus {sbuild dependencies}.

Reply:
Cared.